### PR TITLE
Create userstory-requirement.md

### PR DESCRIPTION
**Pull request by mmccool**
Initial user-story template for discussion.  To be used by other TFs to create databases of user stories that can then be submitted to the UC TF for inclusion in the UC&R document.

### CODE REVIEW

File: REQUIREMENTS/userstory-requirement.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## What: Capability
+- Technical requirement
+- May also be a Condition that needs to be satisfied (e.g. "minimize size")
+- **Link to at least one work item/issue/PR/assertion**
+
+## Why: Purpose
+- Functional requirement
+- Solve a Problem
+- Meet an Objective
+- **Link to at least one Use Case or Use Case Category (e.g. "ease of use")**
```
Comment (mmccool) [inline REQUIREMENTS/userstory-requirement.md:19]:
```suggestion
- **Link to at least one Use Case or Use Case Category (e.g. "ease of use")**
    - Propose a new category if necessary, linking to at least one use case
```

**Comment by mmccool** [comment]
Did some updates based on the discussion in the TD call on Thursday Nov 7, would like to merge and create a YAML issue template based on it.

File: REQUIREMENTS/userstory-requirement.md [hunk 2]
```diff
@@ -1,23 +1,34 @@
 # User Story
-Simple form for user-story based requirements.
+Simple form for user-story based requirements.  While we have split up
+the elements into Who/What/Why each of these should be written as if they fit
+into a sentence template of the following form:
```
Comment (TallTed) [inline REQUIREMENTS/userstory-requirement.md:4]:
```suggestion
Simple form for user-story based requirements.  While we have split 
the elements into Who/What/Why, each of these should be written as if they fit
into a sentence template of the following form:
```

File: REQUIREMENTS/userstory-requirement.md [hunk 3]
```diff
@@ -1,23 +1,34 @@
...
+
+   *As a* Who *I need* What *so I can* Why.
+
+Each slot should be concise, no more than a single line.
+The "What" can also be interpreted as a Technical Requirement
+and the "Why" can be interpreted as a Functional Requirement.
```
Comment (TallTed) [inline REQUIREMENTS/userstory-requirement.md:10]:
```suggestion
Each slot should be concise — no more than a single line.
The "What" can be interpreted as a Technical Requirement,
and the "Why" can be interpreted as a Functional Requirement.
```

File: REQUIREMENTS/userstory-requirement.md [hunk 4]
```diff
@@ -1,23 +1,34 @@
...
 
-## Who: Stakeholder
-- User or Org (e.g. SDO)
+## Who (As a...): Stakeholder
+- User or Org (e.g. SDO) that needs capability
 
-## What: Capability
+## What (I need...): Capability
 - Technical requirement
 - May also be a Condition that needs to be satisfied (e.g. "minimize size")
```
Comment (TallTed) [inline REQUIREMENTS/userstory-requirement.md:20]:
```suggestion
- May also be a Condition that needs to be satisfied (e.g., "minimize size")
```

File: REQUIREMENTS/userstory-requirement.md [hunk 5]
```diff
@@ -1,23 +1,34 @@
...
 - **Link to at least one work item/issue/PR/assertion**
 
-## Why: Purpose
+## Why (so that I can...): Purpose
 - Functional requirement
 - Solve a Problem
 - Meet an Objective
 - **Link to at least one Use Case or Use Case Category (e.g. "ease of use")**
```
Comment (TallTed) [inline REQUIREMENTS/userstory-requirement.md:27]:
```suggestion
- **Link to at least one Use Case or Use Case Category (e.g., "ease of use")**
```

File: REQUIREMENTS/userstory-requirement.md [hunk 6]
```diff
@@ -1,23 +1,34 @@
...
+    - Propose a new category if necessary, linking to at least one use case
 
 ## Details (optional):
-- Free-form extra details as needed
+- Free-form extra details only if needed
```
Comment (TallTed) [inline REQUIREMENTS/userstory-requirement.md:31]:
```suggestion
- Free-form extra details; only if needed
```

File: REQUIREMENTS/userstory-requirement.md [hunk 7]
```diff
@@ -1,23 +1,34 @@
...
+- No more than one paragraph of four sentences.
+- Extra details should be maintained in a separate document that links back this user story
```
Comment (TallTed) [inline REQUIREMENTS/userstory-requirement.md:33]:
```suggestion
- Extra details should be maintained in a separate document that links back to this user story
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by mmccool** [comment]
Will merge after some updates to make it consistent with the new template, but we do have the old requirements-template.md still in the REQUIREMENTS directory and we should do some cleanup and archival.  Let's merge and create and issue to do the cleanup.

**Pull request merged**